resources/views/api/emails/voucher_history_email.blade.php

```diff
@@ -2,6 +2,14 @@
 
 @section('content')
 
-    <p>Voucher History Email</p>
+    <p>Hi there,</p>
+
+    <p>You've requested a record of your voucher payment history, which is attached to this email.</p>
```

Comment:
Technically the user is requesting the history on behalf of a trader. Should we re-word?